### PR TITLE
Add Wait for the Peer Connection to be Established

### PR DESCRIPTION
**Pull request by hardikdr**
# Proposed Changes

Metalbond Peer needs a little bit of wait-time to Establish the connection. 

Fixes #

### CODE REVIEW

File: cmd/metalbond-speaker/main.go [hunk 1]
```diff
@@ -88,6 +90,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	connectionDeadline := time.Now().Add(connectionTimeoutDuration)
+	for time.Now().Before(connectionDeadline) {
```
Comment (afritzler) [inline cmd/metalbond-speaker/main.go:94]: Can you use `wait.PollUntilContect()` instead of using a for loop.

File: cmd/metalbond-speaker/main.go [hunk 2]
```diff
@@ -50,6 +51,7 @@ func main() {
 	var vni int
 	var metalbondServer string
 	var nodeAddress string
+	var connectionTimeoutDuration = 20 * time.Second
```
Comment (afritzler) [inline cmd/metalbond-speaker/main.go:54]: Can we call this metalbond connection timeout or something like that.

**Review by afritzler — CHANGES_REQUESTED** [review]

File: cmd/metalbond-speaker/main.go (hunk 1 above)
Comment (hardikdr) [reply cmd/metalbond-speaker/main.go]: Sure, adapted.

File: cmd/metalbond-speaker/main.go (hunk 2 above)
Comment (hardikdr) [reply cmd/metalbond-speaker/main.go]: Sure, done.

**Review by afritzler — APPROVED** [review]

**Pull request merged**